### ISSUE TITLE
Error while running command exploit: undefined method `cached_size' for nil:NilClass

### ISSUE THREAD

**Issue by 0xffff0800** [rapid7/metasploit-framework#12082]
Trying to run ms17_010_psexec as I was a couple of days ago.. but today its been giving me this error all of a sudden whenever I try to run the exploit. Any ideas? 

```
[-] Error while running command exploit: undefined method `cached_size' for nil:NilClass

Call stack:
/usr/share/metasploit-framework/lib/msf/core/exploit.rb:745:in `is_payload_compatible?'
/usr/share/metasploit-framework/lib/msf/core/exploit.rb:776:in `block in compatible_payloads'
/usr/share/metasploit-framework/lib/msf/core/exploit.rb:775:in `each'
/usr/share/metasploit-framework/lib/msf/core/exploit.rb:775:in `compatible_payloads'
/usr/share/metasploit-framework/lib/msf/ui/console/command_dispatcher/exploit.rb:333:in `choose_payload'
/usr/share/metasploit-framework/lib/msf/ui/console/command_dispatcher/exploit.rb:164:in `cmd_exploit'
/usr/share/metasploit-framework/lib/rex/ui/text/dispatcher_shell.rb:523:in `run_command'
/usr/share/metasploit-framework/lib/rex/ui/text/dispatcher_shell.rb:474:in `block in run_single'
/usr/share/metasploit-framework/lib/rex/ui/text/dispatcher_shell.rb:468:in `each'
/usr/share/metasploit-framework/lib/rex/ui/text/dispatcher_shell.rb:468:in `run_single'
/usr/share/metasploit-framework/lib/rex/ui/text/shell.rb:151:in `run'
/usr/share/metasploit-framework/lib/metasploit/framework/command/console.rb:48:in `start'
/usr/share/metasploit-framework/lib/metasploit/framework/command/base.rb:82:in `start'
/usr/bin/msfconsole:49:in `<main>'
msf5 exploit(windows/smb/ms17_010_psexec) > 
```

**Comment by wvu-r7** [comment]
Dupe of #12076.

**Comment by wvu-r7** [comment]
Symptom fixed in #12085. Will wait to see if the root cause is reproducible.
